docs/install.md

```diff
@@ -37,11 +43,45 @@ services:
       SCREEGO_EXTERNAL_IP: "YOUREXTERNALIP"
 ```
 
+### Port Range
+
 Replace `YOUREXTERNALIP` with your external IP. One way to find your external ip is with ipify.
 ```bash
 $ curl 'https://api.ipify.org'
 ```
```

Comment:
General suggestion: Move this as a information note outside of `Port Range` and `Network Host` to remove the redundancy.